plugins/tiddlywiki/multiwikiserver/modules/store/sql-tiddler-database.js

```diff
@@ -117,6 +214,16 @@ SqlTiddlerDatabase.prototype.createBag = function(bag_name,description,accesscon
 		$accesscontrol: accesscontrol,
 		$description: description
 	});
+	
+
+	// update the permissions on ACL records
```

Comment:
There are still some minor style issues. Comments should be capitalised, and blank lines should not be used so liberally. Generally, we prefer to use comments to delineate sections of code.